resources/js/components/Room/SettingsComponent.vue

```diff
@@ -1,15 +1,13 @@
 <template>
   <div>
     <b-form  @submit.stop.prevent :novalidate="true">
-      <b-overlay :show="saving" >
+      <b-overlay :show="saving || roomTypeSelectBusy" >
```

Comment:
This means on each reload of only the data for the select, the whole form gets layed over?
But the other elements not disabled in this case. Tabbing is still possible! Also I miss a loading overlay on first load. Since when the component gets created a request is sent. But no loading overlay gets shown and if an error occurs a reload would also make sense!

Reply:
I found a solution, I removed to auto. saving and added a save button. Readonly is still missing on all fields. I also added an error message if an error occurred during loading. Tests still missing.